packages/adapter-node/index.js

```diff
@@ -12,10 +12,18 @@ const pipe = promisify(pipeline);
 /**
  * @param {{
  *   out?: string;
- *   precompress?: boolean
+ *   precompress?: boolean;
+ *   env?: {
+ *     host?: string;
+ *     port?: string;
+ *   };
  * }} options
  */
-export default function ({ out = 'build', precompress } = {}) {
+export default function ({
+	out = 'build',
+	precompress,
+	env: { host: hostEnv = 'HOST', port: portEnv = 'PORT' } = {}
```

Comment:
_ahem_ i think you mean `host_env` and `port_env`?

Reply:
Crap, whoops, haha, I do indeed.

Reply:
Fixed in e4d97acbf88533551014cf1a03d19ac700b24098.